docs/reference-docs/rug/rug-conventions.md

```diff
@@ -28,22 +26,23 @@ concrete examples.
 
 [travis-editors]: https://github.com/atomist-rugs/travis-editors
 
-### Rug Archive
+### Rug Projects
```

Comment:
I can see your point but it seems odd to change it out of the blue. Or maybe that was discussed elsewhere?